internal/resources/fetching/fetchers/azure/assets_fetcher_test.go

```diff
@@ -220,7 +220,11 @@ func (s *AzureAssetsFetcherTestSuite) TestFetcher_Fetch() {
 
 			ecs, err := result.GetElasticCommonData()
 			s.Require().NoError(err)
-			s.Empty(ecs)
+			if expected.Type == inventory.VirtualMachineAssetType {
```

Comment:
maybe we can add tests to other cases where the "properties.osProfile.computerName"  nested data is partly/completely missing